### PR TITLE
Improve --help output

### PR DESCRIPTION
**Pull request by cjmayo**
For this PR I've taken the commits from #3242 which only turns the existing output into something more standard, including adding missing `-` in front of options, and adding some extra explanatory text.

This does not document any additional options or change the man page.

### CODE REVIEW

**Review by KjellMorgenstern — APPROVED** [review]
2 minor changes, otherwise looks good to me.

File: src/main.cpp [hunk 1]
```diff
@@ -116,46 +116,50 @@ int main(int argc, char *argv[])
...
 			     "If this is not the case, use the -f option to point to the p/b/s/t folder.\n"
 			     "\n"
+#endif
 			     "The -ep option creates a menu item to launch an external process,\n"
 			     "and puts the standard output of that process into a dialog window in Fritzing.\n"
 			     "The process path follows the -ep argument; the name of the menu item follows the -epname argument;\n"
-			     "and any arguments to pass to the external process are provided in the -eparg arguments.\n";
+			     "and any arguments to pass to the external process are provided in the -eparg arguments.\n"
+			     "\n"
+			     "For more information on Fritzing and its related activities, visit <http://fritzing.org>.\n"
```
[the remark this answers is not in the document]
Comment (cjmayo) [reply src/main.cpp]: Done and for the forum link too.

File: src/main.cpp [hunk 2]
```diff
@@ -116,46 +116,50 @@ int main(int argc, char *argv[])
...
+			     "To report a problem or suggest improvements, use the issue tracker <https://github.com/fritzing/fritzing-app/issues> or the user forum <http://forum.fritzing.org>.\n";
 			break;
 		}
 		case FInitResultVersion: {
```
[the remark this answers is not in the document]
Comment (cjmayo) [reply src/main.cpp]: I haven't found a good way to do this yet. Typically the --help doesn't have versions and I can't see how to add it to --version without it ending up just tacked onto the man page with help2man. It is available in the GUI from Help/About.

**Comment by cjmayo** [comment]
As well as the https changes mentioned above I have made a couple of other changes to:
- Put the "For more information" at the end of the description to stop it being lost at the end of the options.
- Get OPTIONS and REPORTING BUGS sections created by help2man

```diff
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -123,6 +123,10 @@ int main(int argc, char *argv[])
 			     "fostering a creative ecosystem that allows users to document their prototypes, share them with others, "
 			     "teach electronics in a classroom, and layout and manufacture professional PCBs.\n"
 			     "\n"
+			     "For more information on Fritzing and its related activities visit <https://fritzing.org>.\n"
+			     "\n"
+			     "Options:\n"
+			     "\n"
 			     "User options:\n"
 			     "  -d, -debug                    run Fritzing in debug mode, providing additional debug information\n"
 			     //" drc filename : runs a design rule check on the given sketch file\n"
@@ -158,8 +162,7 @@ int main(int argc, char *argv[])
 			     "The process path follows the -ep argument; the name of the menu item follows the -epname argument;\n"
 			     "and any arguments to pass to the external process are provided in the -eparg arguments.\n"
 			     "\n"
-			     "For more information on Fritzing and its related activities, visit <http://fritzing.org>.\n"
-			     "To report a problem or suggest improvements, use the issue tracker <https://github.com/fritzing/fritzing-app/issues> or the user forum <http://forum.fritzing.org>.\n";
+			     "Report bugs or suggest improvements using the issue tracker <https://github.com/fritzing/fritzing-app/issues> or the user forum <https://forum.fritzing.org>.\n";
 			break;
 		}
 		case FInitResultVersion: {
```

**Pull request merged**